### PR TITLE
Lecture/edgar alandete react redux

### PR DESCRIPTION
**Pull request by edalandete**
Today's lecture

### CODE REVIEW

**Review by annasm07 — APPROVED** [review]
Falta test pero prou be, jajaj

File: Edgar-Alandete/lectures/toh-redux/src/Components/HeroDetail/index.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          <button onClick={save} type="button">save</button>
+        </div>
+      )
+      : (
+        <h3>
+          There is no hero with id=
+          {heroId}
+        </h3>
+      )
+
```
Comment (albertmasats) [inline Edgar-Alandete/lectures/toh-redux/src/Components/HeroDetail/index.jsx:64]: This break line is not necessary :)

File: Edgar-Alandete/lectures/toh-redux/src/redux/actions/actionCreators.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+export function loadHero(hero) {
+  return {
+    type: actionTypes.LOAD_HERO,
+    hero,
+  };
+}
+
+export function getHeroById(heroId) {
+  const hero = HEROES.find((current) => current.id === +heroId);
+
```
Comment (albertmasats) [inline Edgar-Alandete/lectures/toh-redux/src/redux/actions/actionCreators.js:40]: Another line break

File: Edgar-Alandete/lectures/toh-redux/src/redux/actions/actionTypes.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export default {
+  LOAD_HEROES: 'LOAD_HEROES',
+  DELETE_HERO: 'DELETE_HERO',
+  ADD_HERO: 'ADD_HERO',
+  UPDATE_HERO: 'UPDATE_HERO',
+
```
Comment (albertmasats) [inline Edgar-Alandete/lectures/toh-redux/src/redux/actions/actionTypes.js:6]: Same.

**Review by albertmasats — APPROVED** [review]

**Review by gteixi — APPROVED** [review]

**Review by gaxxrodri — APPROVED** [review]

**Pull request merged**